api/v1/applicationlayer_types.go

```diff
@@ -27,16 +27,22 @@ type ApplicationLayerSpec struct {
 	WebApplicationFirewall *WAFStatusType `json:"webApplicationFirewall,omitempty"`
 	// Specification for application layer (L7) log collection.
 	LogCollection *LogCollectionSpec `json:"logCollection,omitempty"`
+	// Application Layer Policy controls whether or not ALP enforcement is enabled for the cluster.
+	// When enabled, Services may opt-in to having ingress traffic examined by L7 DaemonSet.
```

Comment:
>  Services may opt-in to having ingress traffic examined by L7 DaemonSet.

Should probably be something like this:

> When enabled, Application-layer ingress policies will be enforced.

Namely, ALP isn't opt-in on the service level (it uses pod selectors) and the fact that this is a DaemonSet is an implementation detail.